api/xcfun.F90.in

```diff
@@ -0,0 +1,298 @@
+!
+! XCFun, an arbitrary order exchange-correlation library
+! Copyright (C) 2018 Ulf Ekström and contributors.
+!
+! This file is part of XCFun.
+!
+! This Source Code Form is subject to the terms of the Mozilla Public
+! License, v. 2.0. If a copy of the MPL was not distributed with this
+! file, You can obtain one at http://mozilla.org/MPL/2.0/.
+!
+! For information on the complete list of contributors to the
+! XCFun library, see: <https://xcfun.readthedocs.io/>
+!
+
+module xcfun
+  implicit none
+!
+  @XCFun_INTEGER@, parameter :: XCFUN_API_VERSION = 2
```

Comment:
I think we should not configure each line but rather configure the kind but this can be done later, no need to delay the PR.

Reply:
Since we set the compiler flags for 64 bit integers, it is indeed not even needed to say `integer(4)` or `integer(8)` here. I prefer not configuring stuff at all, should I remove this?

Reply:
I agree, I think we should remove the configuring of this one.

Reply:
OK, if you can, please post an issue and then I can take it up as a follow-up step.